custom/icds_reports/migrations/sql_templates/database_views/disha_indicators.sql

```diff
@@ -0,0 +1,110 @@
+DROP VIEW IF EXISTS disha_indicators CASCADE;
+CREATE VIEW disha_indicators AS
+SELECT
+    "location_months"."awc_id" AS "awc_id",
```

Comment:
I think that that agg_awc_monthly should include all of these columns, so you could pull these from that view to get rid of a join